rules/app_competition_rules.tex

```diff
@@ -57,6 +57,14 @@ \subsection{Champions Cup and Challenge Shield}
 Teams are assigned to one of the competitions during the application phase, considering their preference, ability and organizational aspects.
 Matches in the champions cup are played with up to 7 players on each team, while the challenge shield is played with up to 5 players on each team (\cf~\cref{sec:robot_players}).
 
+In addition, some rules are simplified for Challenge Shield matches. The major differences are:
+\begin{itemize}
+  \item The requirement to recognize visual referee signals are relaxed for the Champions Cup (\cf~\cref{sec:robot_players}).
```

Comment:
"relaxed for the Champions Cup" seems to suggest that visual ref signals are not as strict in the CC. Did you mean to say that the rules for CS are  "relaxed relative to the Champions Cup" instead?

If so, you should adjust the commented out text in line 65 also

Reply:
Thank you. I wrote the opposite of what I meant by accident. I changed the wording.